pythermalcomfort/models/work_capacity.py

```diff
@@ -0,0 +1,243 @@
+from typing import Union
+
+import numpy as np
+
+from pythermalcomfort.classes_input import (
+    WorkCapacityHothapsInputs,
+    WorkCapacityStandardsInputs,
+)
+from pythermalcomfort.classes_return import WorkCapacity
+
+
+def workcapacity_dunne(wbgt: Union[float, list[float]], intensity: str) -> WorkCapacity:
```

Comment:
please split these functions into separate files, like I am doing for all the other models. I guess you can simply call the model `dunne` no need to add `workcapacity`